modules/Sources/Dependencies/AddressBookClient/AddressBookEncryption.swift

```diff
@@ -150,30 +122,30 @@ extension AddressBookClient {
         var offset = 0
         
         // Deserialize `version`
-        let versionBytes = unencryptedData.subdata(in: offset..<(offset + MemoryLayout<Int>.size))
-        offset += MemoryLayout<Int>.size
+        let versionBytes = try AddressBookClient.subdata(of: unencryptedData, in: offset..<(offset + Constants.int64Size))
```

Comment:
Non-blocking: 

The name `unencryptedData` is less apt now that this can be either data that has never been encrypted, or data that has been decrypted. Maybe change it to just `data`.

Reply:
Updated